docs/conf.py

```
@@ -50,149 +55,250 @@
+    for name in sorted(dir(module)):
+        if name.startswith("_") or name in _SKIP_MEMBERS:
+            continue
 
-# Add any paths that contain custom static files (such as style sheets) here,
-# relative to this directory. They are copied after the builtin static files,
-# so a file named "default.css" will overwrite the builtin "default.css".
-# html_static_path = ["_static"]
+        try:
+            obj = getattr(module, name)
+        except Exception:
+            continue
```

Ruff (0.15.21)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)